client/src/App.tsx

```diff
@@ -401,6 +409,19 @@ function App() {
 		}));
 	}, []);
 
+	const onShowChat = () => {
+		setWaterfallChat((waterfallChat) => ({ ...waterfallChat, show: !waterfallChat.show }));
+	}
+	const updateWaterfallChat = useCallback((message: IMessageEvent) => {
+		const { avatar, value } = message;
+		setWaterfallChat((waterfallChat) => ({ ...waterfallChat, messages: waterfallChat.messages.concat({ "avatar": avatar , "message": value}) }));
+		if(waterfallChat.messages.length > 6){
+			setWaterfallChat((waterfallChat) => (
+					{ ...waterfallChat, messages: waterfallChat.messages.slice(waterfallChat.messages.length - 7 , waterfallChat.messages.length)}
+				));
+		}
+	}, [waterfallChat]);
```

Comment:
dont want to limit messages, instead want a maxHeight and overflowY auto on the messages container